source/change-streams/tests/change-streams.yml

```diff
@@ -2,6 +2,7 @@ collection_name: &collection_name "test"
 database_name: &database_name "change-stream-tests"
 collection2_name: &collection2_name "test2"
 database2_name: &database2_name "change-stream-tests-2"
+collection_rename: &collection_rename "test-rename"
```

Comment:
Hm this requires changes to the spec runner itself, right? I don't see mention of it in the `README`. Is there a way to reuse the `database2`/`collection2` for this purpose (this would require minimal changes in everyones spec runner)? Otherwise, either we need to add a `database3`/`collection3`, or some sort of extensible format (maybe `namespaces: ['test.change-stream-tests', 'test2.change-stream-tests-2'..]`)

Reply:
looks like the actual `collection_rename` is still here, but no longer used anywhere. Safe to remove this now?

Reply:
@mbroadst: GitHub says this thread is outdated. OK to resolve?